src/MohammadYounes.Owin.Security.MixedAuth/MixedAuthHandler.cs

```diff
@@ -58,9 +59,31 @@ public MixedAuthHandler()
                     // Microsoft.Owin.Security.AuthenticationManagerExtensions: ExternalLoginInfo GetExternalLoginInfo(AuthenticateResult result)
                     claimsIdentity.AddClaim(new Claim(ClaimTypes.NameIdentifier, logonUserIdentity.User.Value, null, Options.AuthenticationType));
 
-                    //could grab email from AD and add it to the claims list.
+                    // do we need this ?
                     //claimsIdentity.AddClaim(new Claim(ClaimTypes.WindowsAccountName, logonUserIdentity.Name));
 
+                    // grab email from AD and add it to the claims list.
```

Comment:
I'd rather not to force this specific implementation. For example users may use the `PrincipalContext` as described [here](https://github.com/MohammadYounes/OWIN-MixedAuth/issues/9#issuecomment-82575987).